src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
```

Comment:
I can't change to auto because Lint does not allow to use of more than two attributes per line.
And also for another tag.
Can you help me with this, how can I do this in vs code, maybe there is a good config? @Viktor-Kost 
https://prnt.sc/s3GptGGOd_Wu

Reply:
this happens if you have MORE then two attribute, so if you have 2 it will be ok to place it in one row 
![image](https://github.com/mate-academy/layout_product-cards/assets/96265844/730a45a6-409a-43a9-a7e9-626be89fd086)

Reply:
@maxim2310 
Yes, I understand this, but I thought that any configs can format HTML code automatically because manually it's not very convenient. I tried prettier but it didn't work correctly.